opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.hpp

```diff
@@ -91,15 +92,16 @@ struct GRValState {
 
 public:
     GuideRate(const Schedule& schedule);
-    void   compute(const std::string& wgname, size_t report_step, double sim_time, double oil_pot, double gas_pot, double wat_pot);
-    double get(const std::string& well, Well::GuideRateTarget target, const RateVector& rates) const;
-    double get(const std::string& group, Group::GuideRateProdTarget target, const RateVector& rates) const;
-    double get(const std::string& name, GuideRateModel::Target model_target, const RateVector& rates) const;
-    bool has(const std::string& name) const;
-
+    double getProductionGroupOrWell(const std::string& name, GuideRateModel::Target model_target, const RateVector& rates) const;
+    double getWell(const std::string& well, Well::GuideRateTarget target, const RateVector& rates) const;
+    double getProductionGroup(const std::string& group, Group::GuideRateProdTarget target, const RateVector& rates) const;
+    double getInjectionGroup(const Phase& phase, const std::string& group) const;
```

Comment:
The arguments  should be reordered -  the groupname is the most important argument and should come first.